minard/occupancy.py

```diff
@@ -0,0 +1,97 @@
+from .db import engine_nl
+from .detector_state import get_latest_run
+
+def occupancy_by_trigger(trigger_type, run, find_issues):
+    """
+    Returns a list specifing the normalized occupancy
+    for the trigger type. Pass the find_issues flag
+    to look for issues with the occupancy
+    """
+    conn = engine_nl.connect()
+
+    result = conn.execute("SELECT DISTINCT ON (run, lcn, trigger_bit) "
+                          "lcn, trigger_norm, occupancy "
+                          "FROM trigger_occupancy WHERE trigger_bit = %s AND run = %s "
+                          "ORDER BY run, lcn, trigger_bit ", \
+                          (trigger_type, run))
+
+    rows = result.fetchall()
+    if not rows:
+        return None
+
+    data = [0]*9728
+    trigger_norm = 0
+    for lcn, norm, occupancy in rows:
+        trigger_norm = norm
```

Comment:
Just curious, is this supposed to just be reset every time the loop runs?

Reply:
Yah this is just the same number over the loop so its a bit silly